freestyle-cache-redis/shared/src/test/scala/RedisTests.scala

```diff
@@ -181,14 +181,14 @@ class RedisTests extends AsyncWordSpec with Matchers with RedisTestContext {
     }
 
     "ignore a preceeding PUT on different key" in {
-      def program[F[_] : CacheM]: FreeS[F, Option[Int]] =
+      def program[F[_] : CacheM.To]: FreeS[F, Option[Int]] =
         CacheM[F].put("a", 0) *> CacheM[F].del("b") *> CacheM[F].get("a")
 
       program[CacheM.Op].exec[Future] map { _  shouldBe Some(0) }
     }
 
     "ignore a succeeding PUT on different key" in {
-      def program[F[_] : CacheM]: FreeS[F, Option[Int]] =
+      def program[F[_] : CacheM.To]: FreeS[F, Option[Int]] =
```

Comment:
@raulraja Regarding boilerplate: 

Whenever we want to indicate that, to a type parameter `F : * -> *`, we can put operations from the effect algebra, the type-class is no longer the name of the algebra (no parameters), but the `To` class from its companion object.